test/Test/Indigo/Contracts/HomebaseLite/PropertyTests.hs

```diff
@@ -93,8 +93,8 @@ hprop_main = withTests 500 $ property $ do
       verifyStorage
         :: forall caps m'. MonadEmulated caps m'
```

Comment:
While we're here, this explicit forall is redundant I believe.

Reply:
Removed